src/main/java/nl/praegus/fitnesse/responders/symbolicLink/SymbolicLinkResponder.java

```diff
@@ -0,0 +1,124 @@
+package nl.praegus.fitnesse.responders.symbolicLink;
+
+import fitnesse.FitNesseContext;
+import fitnesse.authentication.SecureOperation;
+import fitnesse.authentication.SecureReadOperation;
+import fitnesse.authentication.SecureResponder;
+import fitnesse.html.template.HtmlPage;
+import fitnesse.html.template.PageTitle;
+import fitnesse.http.Request;
+import fitnesse.http.Response;
+import fitnesse.http.SimpleResponse;
+import fitnesse.wiki.*;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static fitnesse.wiki.WikiPageProperty.SUITES;
+import static nl.praegus.fitnesse.responders.WikiPageHelper.loadPage;
+
+public class SymbolicLinkResponder implements SecureResponder {
+    private PageData pageData;
+
+    @Override
+    public Response makeResponse(FitNesseContext fitNesseContext, Request request) throws Exception {
+        WikiPage wikiPage = loadPage(fitNesseContext, request.getResource(), request.getMap());
+
+        return makeTagResponse(wikiPage, fitNesseContext, request);
+    }
+
+    public SimpleResponse makeTagResponse(WikiPage wikiPage, FitNesseContext fitNesseContext, Request request) throws UnsupportedEncodingException {
+        SimpleResponse response = new SimpleResponse();
+        response.setMaxAge(0);
+        response.setStatus(200);
+        response.setContentType("text/html");
+        response.setContent(createHTMLPage(fitNesseContext, request, wikiPage));
+
+        return response;
+    }
+
+    @Override
+    public SecureOperation getSecureOperation() {
+        return new SecureReadOperation();
+    }
+
+    public JSONArray getSymlink(WikiPage page){
+        return getSymlinkHelper(page, new JSONArray());
+    }
+
+    private JSONArray getSymlinkHelper(WikiPage wikiPage, JSONArray symbolicLinkArray) {
+        pageData = wikiPage.getData();
+        // Get Symlinks of the wiki page
+        WikiPageProperty symLinksProperty = pageData.getProperties().getProperty(SymbolicPage.PROPERTY_NAME);
+
+        // Loop through the names
+        if (symLinksProperty != null) {
+            for (String name : symLinksProperty.keySet()) {
+                symbolicLinkArray.put(getObject(name, wikiPage, symLinksProperty));
+            }
+        }
+
+        // Get Symlinks from the children
+        if (getPageChildren(wikiPage).size() > 0) {
+            for (WikiPage p : getPageChildren(wikiPage)) {
+                getSymlinkHelper(p, symbolicLinkArray);
+            }
+        }
+
+        return symLinksProperty != null ? symbolicLinkArray : new JSONArray();
+    }
+
+    private JSONObject getObject(String name, WikiPage wikiPage, WikiPageProperty symLinksProperty) {
```

Comment:
Maak een nieuw object aan: SymbolicLinkInfo, stop daar de properties in die je nodig hebt, en laat deze functie zo'n object teruggeven. En hernoem deze functie dan als zodanig. Hieronder een beginnetje: 

```
public class SymbolicLinkInfo{

private String pagePath; 

public SymbolicLinkInfo(String pagepath, ... ) {
  this.pagePath = pagePath; 
}

public String getPagePath(){
 return pagePath; 
}


}
```